corehq/apps/reports/filters/forms.py

```diff
@@ -193,7 +193,8 @@ def _translate_name(item):
         map_deleted = sorted(map_deleted, key=lambda item: item['text'].lower())
 
         if (bool(map_deleted) + bool(map_active)) > 1:
-            self.display_app_type = True
+            if self.request.GET.get('show_advanced') != 'on':
+                self.display_app_type = True
```

Comment:
How does this fix the issue?  I thought the problem was that "application type" was defaulting to active even when users selected "show all".  This change looks like it controls whether that dropdown appears at all, but I don't see how it affects the default selection.

Reply:
yeah you are right. But this default only happening to filter level. Result always comes of what filters you select.
Sorry. I applied this check on wrong place. 



Corrected.